eks_fargate/README.md

```diff
@@ -142,8 +140,6 @@ spec:
 
 **Note**: Don't forget to replace `<YOUR_DATADOG_API_KEY>` with the [Datadog API key from your organization][14].
 
-**Note**: Add your desired `kube_cluster_name:<CLUSTER_NAME>` to the list of `DD_TAGS` to ensure your metrics are tagged by your desired cluster. You can append additional tags here as space separated `<KEY>:<VALUE>` tags.
```

Comment:
I would suggest to keep this note for now with an initial sentence like: `prior to agent `7.34.0` or `6.34.0`.